### ISSUE TITLE
unable to get proper tags file for vim

### ISSUE THREAD

**Issue by imiten** [ramitos/jsctags#13]
Hi,  

I tried on bunch of files as well as single file and generate tags file but seems vim will not find the tag.

https://github.com/Countly/countly-server/blob/master/api/api.js

jsctags api.js -f >> tags

Then in vim tried to CTRL + ] with cursor under "validateUserForWriteAPI" on line 248.  It says tag not found.
E426: tag not found: validateUserForWriteAPI

I hope I am attempting the use of this module correctly.

Also to create a tags file for entire dir tree should I do:

find . -name "*.js" | xargs -n 1 -I {} jsctags {} -f >> tags

I tried above and it said tags file not sorted.  cannot find tag.

Regards

Miten.

**Comment by faceleg** [comment]
@mitermayer does this still work? If so do you think it would be a good addition to the README?

**Comment by ianks** [comment]
`find  src -type f -iregex .*\.js$ -exec jsctags {} -f \; | sed '/^$/d' | sort > tags` might be a better option.

**Comment by faceleg** [comment]
@ianks is this worth adding to the README?

**Comment by ianks** [comment]
Yes it would IMO. What would be even better was if the CLI had a better interface, though.

**Comment by faceleg** [comment]
PR!
